### PR TITLE
S 55/el 2383

### PR DESCRIPTION
**Pull request by AlastairMcKee**
Migrating components tabs - wizard

### CODE REVIEW

**Comment by ashley-hunter** [comment]
Conflicts need resolved

**Comment by AlastairMcKee** [comment]
Conflicts resolved

File: docs/app/pages/components/sections/tabs/card-tabs-ng1/card-tabs-ng1.component.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { Component } from '@angular/core';
+import { DocumentationSectionComponent } from '../../../../../decorators/documentation-section-component';
+import { ICodePenProvider } from '../../../../../interfaces/ICodePenProvider';
+import { ICodePen } from '../../../../../interfaces/ICodePen';
+
+@Component({
+    selector: 'uxd-components-card-tabs-ng1',
```
Comment (ashley-hunter) [inline docs/app/pages/components/sections/tabs/card-tabs-ng1/card-tabs-ng1.component.ts]: Most sections should be able to specify a change detection strategy of OnPush to give us a bit better page performance

File: docs/app/pages/components/sections/tabs/detailed-tab-example-ng1/detailed-tab-example-ng1.component.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+<hr>
+
+<p>Above is a more detailed example of how the tabs can be used within an application, its is recommended to follow this design to avoid errors.</p>
+
+<tabset>
+    <tab heading="HTML">
+        <uxd-snippet [code]="htmlCode"></uxd-snippet>
+    </tab>
+    <tab heading="Ctrl">
```
Comment (ashley-hunter) [inline docs/app/pages/components/sections/tabs/detailed-tab-example-ng1/detailed-tab-example-ng1.component.html]: Change heading to Javascript to match other sections

**Comment by AlastairMcKee** [comment]
Made review changes & added an on destroy function to the static tooltips as the tooltip was still visible after changing pages.

File: src/ng1/directives/staticTooltip/staticTooltip.controller.js [hunk 3]
```diff
@@ -206,6 +206,10 @@ export default function StaticTooltipCtrl($scope, $element, $parse, safeAnimatio
     function(){ 
         $staticTooltip.destroyAllTooltips();
     });
+
+    $scope.$on('$destroy', function() {
+        $staticTooltip.destroyAllTooltips();
```
Comment (ashley-hunter) [inline src/ng1/directives/staticTooltip/staticTooltip.controller.js]: This should probably only destroy the tooltip bound to this scope, we probably shouldn't remove all tooltips whenever the scope of one is destroyed. I may be wrong, but check to see if only destroying the tooltip bound to that scope works

**Comment by AlastairMcKee** [comment]
Made change to only destroy the bound tooltip

**Comment by ashley-hunter** [comment]
Looks good, tslint passes

**Pull request merged**